doc/keynames/README.md

```diff
@@ -0,0 +1,213 @@
+# Key Names in Elektra
```

Comment:
@markus2330 @mpranj I wasn't quite sure where to put this (and the accompanying Python file), so I made a new directory for now. If you have better ideas, let me know...